examples/SkyTEM1D/Menindee/read_survey_data.jl

```diff
@@ -0,0 +1,177 @@
+#Steps to read and modify the SKyTEM headerfile 
+#module to read the hdr file and its column number 
+using DataFrames
+using CSV
+using DelimitedFiles
+using Statistics
+using PyPlot
+
+function read_survey_files(hdrfile::String,fname_dat::String,fname_specs_halt::String;
```

Comment:
This function is now defined in `SkyTEM1DInversion.jl`, so should we be using the version from there to run the example scripts?